tests/installation/bootloader_uefi.pm

```diff
@@ -53,7 +53,8 @@ sub run {
     # Press key 't' to let grub2 boot menu show up in serial console
     if (is_ipmi && current_console eq 'sol' && is_selfinstall && get_var('IPXE_UEFI')) {
         assert_screen('press-t-for-boot-menu', 180);
-        send_key('t');
+        sleep 5;
```

Comment:
Why sleeping 5 secs here ?

Reply:
I think you can try typing ```t``` multiple times, for example ```send_key('t') for (0..2);``` instead of sleeping.

Reply:
Tried ever, but not work,